Sources/_FoundationInternals/String/String+Internals.swift

```diff
@@ -0,0 +1,19 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
```

Comment:
```suggestion
// Copyright (c) 2023 Apple Inc. and the Swift project authors
```

Reply:
I would like if a robot could remind me every time I forget to update the copyright year.

Reply:
We will need some sort of linting soon. We can def add that.